### PR TITLE
Updated Safari Bug

### PR DESCRIPTION
**Pull request by michael-letcher**
Safari bug is no longer open and has been resolved. Changing Docs to reflect this.

### CODE REVIEW

**Comment by XhmikosR** [comment]
In which version was this fixed? Has it been released?

**Comment by Herst** [comment]
@XhmikosR In 10.1 apparently, see <https://github.com/philipwalton/flexbugs#11-min-and-max-size-declarations-are-ignored-when-wrapping-flex-items>

File: docs/4.0/layout/grid.md [hunk 1]
```diff
@@ -149,9 +149,9 @@ For example, here are two grid layouts that apply to every device and viewport,
 {% endexample %}
 </div>
 
-Equal-width columns can be broken into multiple lines, but there is a [Safari flexbox bug](https://github.com/philipwalton/flexbugs#11-min-and-max-size-declarations-are-ignored-when-wrapping-flex-items) that prevents this from working without an explicit `flex-basis` or `border`. Our example works thanks to the `border` being set; you can do the same with `.col { border: 1px solid transparent; }`. Alternatively, you can set the flex-basis to the width of the column (e.g., `.col { flex: 1 0 50%; }`).
+Equal-width columns can be broken into multiple lines, but there was a [Safari flexbox bug](https://github.com/philipwalton/flexbugs#11-min-and-max-size-declarations-are-ignored-when-wrapping-flex-items) that prevented this from working without an explicit `flex-basis` or `border`.
 
-Both these fixes have been documented in a [reduced test case outside Bootstrap](https://output.jsbin.com/micohor).
+Two workarounds have been documented in a [reduced test case outside Bootstrap](https://output.jsbin.com/micohor), though if the browser if up to date this shouldn't be necessary.
```
Comment (gijsbotje) [inline docs/4.0/layout/grid.md]: you've got a typo on this line
`though if the browser if up to date`

**Comment by XhmikosR** [comment]
@michael-letcher  has deleted his fork. You could have waited until this was merged...

**Comment by michael-letcher** [comment]
@XhmikosR Sorry about that, I didn't realize I had.

**Pull request closed without merging**